frontend/common/kubeflow-common-lib/projects/kubeflow/src/lib/form/rok-url-input/rok-url-input.component.scss

```diff
@@ -19,7 +19,7 @@
     background: none;
     cursor: default;
   }
-
+  /* TODO(mdc-migration): The following rule targets internal classes of form-field that may no longer apply for the MDC version. */
```

Comment:
Is there an issue for this?

Reply:
Its from the automated script to migrate the legacy angular components https://jirab.statcan.ca/browse/BTIS-84 . But its in an "rok" component, which is not a part of the repo that we actually use. So there's no real need to update this. (Honestly the entire rok stuff should just get removed)